src/docs/nation.mdx

```diff
@@ -11,33 +11,33 @@ custom_edit_url: https://github.com/EarthMC/earthmc-docs/blob/main/src/docs/nati
 
 :::
 
 ## Nation basics
-As king of your nation, you can add allies with `/n ally add` and invite towns to your nation with `/n add`. Towns added must be in proximity of 3000 blocks from the capital to be added.
+As a leader of your nation, you can add allies with `/n ally add nation` and invite towns to your nation with `/n add town`. Added towns must be in distance of 3000 blocks from the capital.
 
-Give and remove ranks from citizens of your nation with `/n rank add username rank` or `/n rank remove username rank`. You can also kick towns from the nation with `/n kick`, delete the nation with `/n delete`, toggle public joining to your nation with `/n toggle open`, and set the nation’s spawn location anywhere within your capital with `/n set spawn`.
+Give and remove ranks from residents of your nation with `/n rank add player rank` or `/n rank remove player rank`. You can also kick towns from the nation with `/n kick town`, delete the nation with `/n delete`, toggle public joining to your nation with `/n toggle open`, and set the nation’s spawn location anywhere within your capital with `/n set spawn`.
 
-Due to nations starting with no allies, you will likely want to get a lot as quickly as you can as you require them to travel anywhere with the `/n spawn` command. An efficient way to do so is by regularly checking `/res list`, and running the `/res` command on all nation kings as marked by a yellow name, then do `/t town_name` for the town listed in their `/res` and send an ally request to them.
+Due to nations starting with no allies, you will likely want to get a lot as quickly as you can as you require them to travel anywhere with the `/n spawn` command. An efficient way to do so is by regularly checking `/res list`, and running the `/res` command on all nation kings as marked by a yellow name, then do `/t name` for the town listed in their `/res` and send an ally request to them.
 
-As your nation gains more citizens, your ranking on `/n list` will increase. Nations fight each other to be on the top of this list as it shows who has put the most time into recruiting towns and players, which is a good indicator of activity. Nations also compete to have the most townblocks in their nation. This can be seen on `/n list by townblocks`.
+As your nation gains more citizens, your ranking on `/n list` will increase. Nations fight each other to be on the top of this list as it shows who has put the most time into recruiting towns and players, which is a good indicator of activity. Nations (as well as towns) can also compete in a few categories. These leaderboards can be seen via `/nation|town list by category` (categories should be automatically shown to the command sender).
 
 ## Nation ranks
-Nation kings can give nation residents ranks with `/n rank add {username} {rank}`.
+Nation kings can give nation residents ranks with `/n rank add player rank`.
```

Comment:
The user input command parameters should still be marked separately from the static ones. Applies to all commands in the docs.